### PR TITLE
dashboards: update cluster dashboard

### PR DESCRIPTION
**Pull request by hagen1778**
* add assisted merges panel https://github.com/VictoriaMetrics/VictoriaMetrics/issues/2754
* add mem panel per each component
* remove lines filling for some panels for clarity

Signed-off-by: hagen1778 <roman@victoriametrics.com>

image
image

### CODE REVIEW

File: dashboards/victoriametrics.json [hunk 1]
```diff
@@ -6802,8 +7282,8 @@
       {
         "current": {
           "selected": false,
-          "text": "VictoriaMetrics",
-          "value": "VictoriaMetrics"
+          "text": "1015",
+          "value": "1015"
```
Comment (k1rk) [inline dashboards/victoriametrics.json:7286]: lets keep this unchanged =)

Comment (hagen1778) [inline dashboards/victoriametrics.json:7286]:
```suggestion
          "text": "VictoriaMetrics",
          "value": "VictoriaMetrics"
```

Comment (hagen1778) [reply dashboards/victoriametrics.json]: updated

**Review by k1rk — APPROVED** [review]

**Review by f41gh7 — APPROVED** [review]
LGTM

**Pull request merged**